srcpkgs/OpenRCT2/INSTALL.msg

```diff
@@ -3,3 +3,6 @@ You must install a package that provides either the zenity or kdialog binaries:
 - zenity
 - qarma
 - kdialog
+
+OpenRCT2 requires the original RollerCoaster Tycoon 2 game files to function;
+visit https://openrct2.io/getting-started/index.html for details.
```

Comment:
I don't love the idea of adding more the msg. Do you think it could be split into a README.voidlinux or something?

Reply:
This is referenced in all the associated documentation for the game, so I don't know if a Void-specific README makes sense. The message is intended for users who do a drive-by installation after searching the Void repos without looking up the game separately; if we had an "extended description" field, I would have added it there instead. I would like to get rid of the other section of the message though, that really should be a vpkg or something.

I don't have a strong feeling about the added message though, so if you think it would be better off without, I'd be fine removing it.

Reply:
Okay, INSTALL.msg yeeted and zenity dep added